src/main/java/appeng/tile/misc/TileCondenser.java

```diff
@@ -342,15 +344,17 @@ public IStorageMonitorable getInventory( IActionSource src )
 		}
 
 		@Override
-		public IMEMonitor<IAEItemStack> getItemInventory()
+		public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEMonitor<T> getInventory( IStorageChannel<T> channel )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') C is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2326)